src/lib/clock.js

```diff
@@ -0,0 +1,14 @@
+import moment from 'moment-timezone';
+
+//initialization code
+setTimeout(() => {
+    //TODO: connect email system here
+    //email Students
+    //setInterval(/*Email Students*/, 86400000);
+}, (moment.tz(Date.now(), "America/Chicago").hour(16).minute(30).second(0).format('X')
+ - moment.tz(Date.now(), "America/Chicago").format('X')) * 1000);
+
+export function isBeforeNow(timestamp){
+    let currentTime = moment.tz(Date.now(), "America/Chicago");
+    return timestamp < parseInt(currentTime.format('X'));
```

Comment:
At the time I couldn't remember if it was just 1000 to divide. That's why I converted it the way I did. Will change this.